x-pack/plugins/security_solution/public/detection_engine/rule_management/api/api.ts

```diff
@@ -698,3 +707,56 @@ export const bootstrapPrebuiltRules = async (): Promise<BootstrapPrebuiltRulesRe
     method: 'POST',
     version: '1',
   });
+
+/**
+ * SIEM Rules Migration routes
+ */
+
+/**
+ * Retrieves the stats for all the existing migrations, aggregated by `migration_id`.
+ *
+ * @param signal AbortSignal for cancelling request
+ *
+ * @throws An error if response is not OK
+ */
+export const getRuleMigrationsStatsAll = async ({
```

Comment:
These new functions are not part of the rule_management workflow, but siem rule migrations results instead. Let's try to decouple as much as possible from rules management. Can we create our own `api.ts` lib?